index.md

```diff
@@ -143,16 +143,14 @@ The official Black Magic Probe hardware is available from these distributors in
  * USA: [Adafruit Industries](https://www.adafruit.com/product/3839) Black Magic Probe with JTAG Cable and Serial Cable - V2.3 (Newest)
 
 ## Other Hardware supported by Black Magic Debug:
- * Stlink V2.1 and V3 based boards with a BMDA variant of the BMD (`PROBE_HOST=pc-stlinkv2`)
- * LibFTDI based based boards boards with a BMDA variant of the BMD (`PROBE_HOST=libftdi`)
- * StlinkV2 based boards with BMD ST-Link firmware (`PROBE_HOST=stlink ST_BOOTLOADER=1`) flashed with [stlink-tool](https://github.com/jeanthom/stlink-tool.git) to unmodified board, see [Readme.md](https://github.com/blacksphere/blackmagic/tree/master/src/platforms/stlink).
- * StlinkV2 based boards with BMD ST-Link firmware (`PROBE_HOST=stlink`) with BMD own bootloader flashed to the board.
- * STM8S Discovery (StlinkV1) and Bluepill board with BMD SW-Link firmware (`PROBE_HOST=swlink`)
- * F4 Discovery (STM32F407) with BMD f4discovery firmware (`PROBE_HOST=f4discovery`)
- * Blackpill V2 (STM32F411) (`PROBE_HOST=blackpillv2`)
- * launchpad-icdi with BMD launchpad-icdi firmware (`PROBE_HOST=launchpad-icdi`)
- * Hydrabus(?) (`PROBE_HOST=hydrabus`)
- * TM4C(?) (`PROBE_HOST=tm4c`)
+ * ST-Link V2, V2.1. See [stlink](https://github.com/blackmagic-debug/blackmagic/tree/master/src/platforms/stlink)
```

Comment:
This platform is actually the one for Bluepill. There's only a very particular set of "bluepills" that use the swlink platform.

Reply:
That surprises me, as on GitHub the [stlink readme](https://github.com/blackmagic-debug/blackmagic/tree/main/src/platforms/stlink) does not contain any mention of Blue Pill, whereas the [swlink readme](https://github.com/blackmagic-debug/blackmagic/tree/main/src/platforms/swlink) mentions Blue Pill explicitly.

Also the `hardware.md` page mentions that `swlink` can be used for Blue Pill boards.

On GitHub I indeed find `BLUEPILL` in the Makefile of `stlink`. However, if `stlink` is the platform that is supposed to be used as default to compile for a Blue Pill, the readme's on GitHub and the hardware page on the website are misleading.

I can update the index and hardware page to mention that by default the `stlink` platform shall be used, and only for a particular set of Blue Pills the `swlink` platform shall be used. Can you inform me which "particular set" shall use the `swlink` platform, such that I can include this information directly on the website?

Reply:
This is because the README for the `stlink` platform is wrong and we've been noodling on how to change and update it.

To use the firmware on a normal Bluepill, the build line is `make PROBE_HOST=stlink BLUEPILL=1`. There are some "bluepills" (as people insist on calling them - we're unsure if they're actually the WeAct Studio Bluepill board, or something else) which do instead use the swlink platform, but we've been as-yet unable to figure out when and why which has been the problem in wanting to update the instructions and correct the mistakes. The previous maintainer left us far too little information written down about this.

If you'd like to have a crack at updating those READMEs, it would be a welcome addition along side this PR.

Reply:
I have updated the index and hardware page to make clear that the `stlink` platform with `BLUEPILL=1` shall be used for the Blue Pill boards.